src/nti/fakestatsd/__init__.py

```diff
@@ -22,24 +22,27 @@ def type(self):
 class Gauge(AbstractMetric):
     kind = 'g'
 
-
 class Counter(AbstractMetric):
     kind = 'c'
 
+class Set(AbstractMetric):
+    kind = 's'
 
 class Timer(AbstractMetric):
-
-    def __init__(self, name, value, sampling_rate, kind):
-        super(Timer, self).__init__(name, value, sampling_rate)
-        self.kind = kind
+    kind = 'ms'
 
 METRIC_TYPES = {
     'g': Gauge,
     'c': Counter,
-    's': Timer,
+    's': Set,
     'ms': Timer
 }
 
+def _parse_sampling_data(data):
+    if not data.startswith('@'):
+        raise ValueError('Expected "@" in sampling data. %s', data)
```

Comment:
Did you mean to give the constructor two arguments, or is a `%` operator missing?

Reply:
Thanks.